net/pfSense-pkg-siproxd/files/usr/local/pkg/siproxd.xml

```diff
@@ -516,6 +516,29 @@
 			<type>input</type>
 			<default_value>192.168.0.0/16,10.0.0.0/8,172.16.0.0/20</default_value>
 		</field>
+		<field>
+			<name>Plugin Settings - SIP Trunk</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>SIP Trunk</fielddescr>
+			<fieldname>plugin_siptrunk</fieldname>
+			<description>Enable SIP Trunk Plugin</description>
+			<sethelp>Plugin to handle SIP Trunks where using *one* single SIP account a whole number block is routed. This means asn incoming INVITE does carry the target number (in SIP URI or To: header field) but does not really carry any clear indications to which account it belongs to.</sethelp>
+			<type>checkbox</type>
+		</field>
+		<field>
+			<fielddescr>SIP account</fielddescr>
+			<fieldname>plugin_siptrunk_account</fieldname>
+			<description>SIP account in the form of 'sip:user@host', identical as used for registration.</description>
+			<type>input</type>
+		</field>
+		<field>
+			<fielddescr>REGEX number</fielddescr>
+			<fieldname>plugin_siptrunk_numbers_regex</fieldname>
+			<description>Regular expression that matches the whole number block associated with this account</description>
```

Comment:
Would be nice to have an example here to show the expected format of the regex (since parsers vary)

Reply:
done